Design.md

```diff
@@ -0,0 +1,52 @@
+# Design
+ 
+## Senior UI/UX Designer
+- I have at least 5 years experience
+- I display initiative and ownership
+
+- I can lead a project day to day (Design Lead)
+    - I can make strategic design and UX decisions about both core and new functions and features
+    - I can measure and capture user feedback on a regular basis.
+- I can create design guidelines to set the tone and design rules through out a project life-cycle
+- I can oversee the user experience from conception to launch in partnership with product owners and developers
+- I am responsible for overall team performance and building a culture of design excellence
+- I am comfortable with running the Scrum ceremonies (i.e. can act as Scrum Master)
+- I am comfortable with mentoring/teaching other designers 
+- I can define the user experience best practice for client porjects and internal projects
+
+
+##  UI/UX Designer
+- I have at least 3 years experience
+- I have Excellent visual design skills with an eye for detail
+
+- I can build/gather requirements from client conversations
+    - I can assist on Specification Reviews
+    - I can do initial meetings with clients
+    - I can creates the design of user interface specifications, by applying client research and design guidelines
+- I can design user interfaces that encompass an end-to-end experience across a variety of platforms
+- I can represent work to clients and other team members, clearly and succinctly articulating the goals and design concepts
+- I can excute the UX process from end to end, including wireframes, behavioral specifications, and high-fidelity prototypes
+- I take collaborative approach to working with engineering and product teams
+- I am comfortable with all facets of the Scrum process 
+    - I provide suggestions in Scrum reviews/retros
+    - I break PBIs down in planning
+    - I can estimate work to be done
+    - I help the Product Owner and Scrum Master proactively (i.e. give suggestions/recommendations)
+- I provide effective feedback internally
+
+
+## Design Work Experience
+- I have approximately 1 year of experience in graphic or user interface design (in industry or otherwise)
+- or I may be just out of University and/or still studying for a design related degree
+- I have good communication skills
+
+- I can confidently deliver PBIs for clients
+    - I can interpret a PBI from the backlog
+    - I can break it down into Tasks
+    - I can complete the Tasks and confidently demonstrate them
+- I am comfortable with the Scrum process 
+    - I can talk about my work in a Daily Scrum
+    - I speak up in Sprint Reviews and Retrospectives
+- I ask for help when I'm blocked
+- I can take on feedback and improve
+- I possess knowledge and skill in design softwares (e.g. Adobe Suite, Sketch and Figma)
```

Comment:
I believe the plural for software is software :)

Reply:
Done